Unix/installbuilder/conf/omiserver.conf

```diff
@@ -58,6 +60,3 @@ keyfile=/etc/opt/omi/ssl/omikey.pem
 configfile=/etc/opt/omi/conf/omiserver.conf
 #NoSSLv2=true
```

Comment:
61-62 work, I know because I added them. But they should be in a separate security section or something explaining what they do. I think it's clear from the name, but a line or two of explanation might be good. In general, we disable SSLv2 because it's not considered secure.